src/utils/formHelperFunctions.ts

```diff
@@ -76,6 +78,23 @@ export function getDefaultFieldValue(field: any) {
   }
 }
 
+export function getNonHiddenFields(currentStep: any, formattedFields: any) {
+  const hiddenState: { key: string; hidden: boolean }[] =
+    currentStep.servar_fields.map((servar: any) => ({
+      key: servar.servar.key,
+      hidden: shouldElementHide({ element: servar })
```

Comment:
cleaner to just call .filter and run the shouldElementHide function. check out how we use this function in other places